include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -322,6 +337,106 @@ class argument_parser
+ This ensures you will be informed about new versions and important
+ fixes and is highly recommended. It involves making a web-request
+ to a server and transmitting basic application metadata (of course
+ not your data files!). For privacy implications, please see:
+ https://github.com/seqan/seqan3/wiki/Update-Notifications
+
+    [a] Always perform version checks for this app (the default).
+    [n] Never perform version checks for this app.
+    [y] Yes, perform a version check now, and ask again next time.
+    [s] Skip the version check now, but ask again next time.
+
+ You can change the behaviour at any time by passing
+      --version-check X
+ to your application where "X" is one of the above letters or you can
+ permanently disable the version check by setting the environment variable
+ SEQAN3_NO_VERSION_CHECK.
+
+#######################################################################
+Please enter one of [a, n, y, s]: )";
+
+                        while (!version_check.has_value())
+                        {
+                            std::string answer;
+                            std::cin >> answer;
+
+                            if (answer.size() != 1)
+                            {
+                                std::cerr << "Please enter one of [a, n, y, s]: ";
+                                continue;
+                            }
+
+                            switch (answer[0])
+                            {
+                                case 'y':
+                                {
+                                    version_check = true;
+                                    break;
+                                }
+                                case 's':
+                                {
+                                    version_check = false;
+                                    break;
+                                }
+                                case 'n':
+                                {
+                                    version_check = false;
+                                    app_version.write_cookie(std::string{"NEVER"});
+                                    break;
+                                }
+                                case 'a':
+                                {
+                                    version_check = true;
+                                    app_version.write_cookie(std::string{"ALWAYS"});
+                                    break;
+                                }
+                                default:
+                                {
+                                    std::cerr << "Please enter one of [a, n, y, s]: ";
+                                }
+                            }
+                        }
+                    }
+                    else // if !detail::is_terminal()
+                    {
+                        version_check = true; // default: check version if you cannot ask the user
```

Comment:
You still need to display something like in SeqAn2...

Reply:
What do you mean? 

The messages "there is a new version..." are printed when the version check is performed `operator()` of the `version_checker` (where I ensure that it is only printed once a day).